src/codeu/chat/client/core/Controller.java

```diff
@@ -40,6 +40,94 @@ public Controller(ConnectionSource source) {
   }
 
   @Override
+  public String newStatusUpdate(Uuid user) {
+
+    String statusUpdate = null;
+    try (final Connection connection = source.connect()) {
+      Serializers.INTEGER.write(connection.out(), NetworkCode.NEW_STATUS_UPDATE_REQUEST);
+      Uuid.SERIALIZER.write(connection.out(), user);
+
+      if (Serializers.INTEGER.read(connection.in()) == NetworkCode.NEW_STATUS_UPDATE_RESPONSE) {
+        statusUpdate = Serializers.STRING.read(connection.in());
+      } else {
+        LOG.error("Response from server failed.");
+      }
+
+    } catch (Exception ex) {
+      System.out.println("ERROR: Exception during call on server. Check log for details.");
+      LOG.error(ex, "Exception during call on server.");
+    }
+
+    return statusUpdate;
+  }
+
+  @Override
+  public void unfollowUser(User userA, User userB) {
+
+    try (final Connection connection = source.connect()) {
+      Serializers.INTEGER.write(connection.out(), NetworkCode.NEW_UNFOLLOW_USER_REQUEST);
+      User.SERIALIZER.write(connection.out(), userA);
+      User.SERIALIZER.write(connection.out(), userB);
+
+      if (!(Serializers.INTEGER.read(connection.in()) == NetworkCode.NEW_UNFOLLOW_CONVERSATION_RESPONSE)) {
+        LOG.error("Response from server failed.");
+      }
+    } catch (Exception ex) {
+      System.out.println("ERROR: Exception during call on server. Check log for details.");
+      LOG.error(ex, "Exception during call on server.");
+    }
+  }
+
+  @Override
+  public void followUser(User userA, User userB) {
+
+    try (final Connection connection = source.connect()) {
+      Serializers.INTEGER.write(connection.out(), NetworkCode.NEW_FOLLOW_USER_REQUEST);
+      User.SERIALIZER.write(connection.out(), userA);
```

Comment:
UUID makes sense for these as well since you don't care about anything except the unique identifier for the users.